models/boost/arbitrum/boost_arbitrum_claimed.sql

```diff
@@ -0,0 +1,45 @@
+ {{
+    config(
```

Comment:
please refer to [models docs](https://github.com/duneanalytics/spellbook/blob/main/docs/models/model_config_block.md) for required properties in config blocks, depending on materialization type

with this current setup, these will all default to views (which is fine, if performance on the app downstream is good for you). however, without `alias` applied, i believe dbt will use the file name as the alias. with that in mind, in the current state, this spell would build as:
`boost_arbitrum.boost_arbitrum_claimed`
where i'd imagine you'd prefer
`boost_arbitrum.claimed`
which can be accomplished with alias property

the attached CI job builds out custom naming for us to easily differentiate in backend. here is example:
`test_schema.git_dunesql_64b9f6c_boost_arbitrum_deployed`
(which you can query on dune btw, until a daily job cleans up all CI tables, then you can simply rebuild if needed)

Reply:
you can add both `alias` and `materialized` in all chain-level spells in this PR

Reply:
added `alias` to all models and `materialized` to all-chain level models.